### PR TITLE
Make it possible to use an external DHCP

### PR DESCRIPTION
**Pull request by haslersn**
On the bridge network, client IP address allocation can be handled in two ways:

* If `enable_dhcp` is `true`: Let OpenVPN run an own DHCP server on the bridge network.
* If `enable_dhcp` is `false`: Use an external DHCP server that's connected through the bridged ethernet interface. You have to set that DHCP server up yourself.

### CODE REVIEW

**Comment by haslersn** [comment]
This is now tested to work.

File: templates/up.sh.j2 [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 if [ $? != 0 ]; then
 	sleep 2
 	exec systemctl restart openvpn.service
 fi
-brctl addif {{ openvpn_bridge_dev }} {{ openvpn_dev }}; true
-brctl addif {{ openvpn_bridge_dev }} {{ openvpn_interface }}; true
-ip link set {{ openvpn_bridge_dev }} up;
-ip link set {{ openvpn_dev }} up;
-ip address replace {{ openvpn_bridge.address }} dev {{ openvpn_bridge_dev }}; true
+brctl addif {{ openvpn_bridge_name }} tap0; true
```
Comment (Mr-Pi) [inline templates/up.sh.j2]: Please use the openvpn environment or argument variables

File: templates/up.sh.j2 [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+brctl addif {{ openvpn_bridge_name }} {{ openvpn_bridge_eth_interface }}; true
+ip link set {{ openvpn_bridge_name }} up;
```
Comment (Mr-Pi) [inline templates/up.sh.j2]: Please add a fallback for ifconfig, when iproute2 is not available.

File: templates/up.sh.j2 (hunk 1 above)
Comment (haslersn) [reply templates/up.sh.j2]: Done.

**Pull request merged**